app/CocoaPods/CLI Integrations/CPCLIToolInstallationController.m

```diff
@@ -358,4 +415,63 @@ - (BOOL)installBinstubToPrivilegedDestination;
   return succeeded;
 }
 
+#pragma mark - Binstub uninstallation
+
+/// Loops through all installed destinations and tries to remove them.
+///
+/// @return Dictionary of remaining bookmarks which couldn't be removed. Empty dict means everything was succuessfully removed.
+///
+- (NSDictionary *)removeBinstubAccordingToPrivileges
+{
+  self.errorMessage = nil;
+  
+  NSFileManager *fileManager = [NSFileManager defaultManager];
+  
+  NSMutableDictionary *URLs = [self.previouslyInstalledToDestinations mutableCopy];
+  for (NSURL *url in self.previouslyInstalledToDestinations) {
+    
+    if (![fileManager fileExistsAtPath:url.path]) {
+      // remove url from our list
+      [URLs removeObjectForKey:url];
+      continue;
+    }
+    
+    NSLog(@"Removing binstub: %@", url.path);
+    
+    BOOL removed = NO;
+    if ([self hasWriteAccessToBinstubWithURL:url]) {
+      removed = [self removeBinstubFromAccessibleDestinationWithURL:url];
+    } else {
+      removed = [self removeBinstubFromPrivilegedDestinationWithURL:url];
+    }
+    
+    if (removed) {
+      [URLs removeObjectForKey:url];
+    }
+  }
+  return [URLs copy];
+}
+
+- (BOOL)removeBinstubFromAccessibleDestinationWithURL:(NSURL *)url
+{
+  NSError *error = nil;
+  BOOL success = [[NSFileManager defaultManager] removeItemAtURL:url error:&error];
+  
+  if (error) {
+    NSLog(@"Failed to remove binstub: %@", error);
+    self.errorMessage = @"Failed to remove pod command";
+    success = NO;
+  }
+  
+  return success;
+}
+
+- (BOOL)removeBinstubFromPrivilegedDestinationWithURL:(NSURL *)url
+{
+  
+  // TODO: how to do this?!?
```

Comment:
Maybe do it in the same way as instelling it, but instead invoke the 'rm' CLI tool?